docs/articles/output.html

```diff
@@ -544,12 +544,12 @@ <h2 id="exclusions-generated-by-adult-algorithm">Exclusions generated by adult a
 
       <footer><div class="copyright">
   <p></p>
-<p>Developed by Daymont Carrie, Grundmeier Robert, Miller Jeffrey, Campos Diego.</p>
+<p>Developed by Carrie Daymont, Robert Grundmeier, Jeffrey Miller, Diego Campos.</p>
 </div>
 
 <div class="pkgdown">
   <p></p>
-<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
+<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
```

Comment:
Huh, this went backwards?

Reply:
My pkgdown must not be up to date! I'll update and rebuild.